transcom/mymove
# B 17814 Removing references to Fort Benning and removing incorrect link between Fort Walker and Fort Benning.

Removing references to Fort Benning and removing incorrect link between Fort Walker and Fort Benning

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/TeamRoom.mvc/Show/808731)

## Summary
This PR addresses issues I-12142 and I-12141, written up against B 17814.

Fort Benning was renamed to Fort Moore. Fort A.P. Hill was renamed to Fort Walker. There was a link in the database between Fort Benning and Fort Walker, which is incorrect. The zip codes of 31905 and 31995 should be associated with Fort Moore, GA, instead of Fort Walker, GA.

This also contains a rename of Blackstone, VA in the duty locations table to Fort Barfoot so that when users type in Fort Barfoot, they can see Fort Barfoot in the search results with the address of Blackstone, VA populated under the duty location bar.

## Verification Steps for the Author


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the milmove application
2. Login as a customer starting a new move.
3. Type in Fort Barfoot and you should see Fort Barfoot in the duty location line and underneath it, Blackstone, VA 23824.
4. Type in Ft Benning and no Fort Benning values should display.
5. Type in Ft Walker and it should display Fort Walker, VA 22427.

### Database

#### Any new migrations/schema changes:


## Screenshots
